lib/working_hours/computation.rb

```diff
@@ -85,6 +85,26 @@ def advance_to_working_time time, config: nil
       end
     end
 
+    def advance_to_closing_time time, config: nil
+      config ||= wh_config
+      time = in_config_zone(time, config: config).round
+      loop do
+        # skip holidays and weekends
+        while not working_day?(time, config: config)
+          time = (time + 1.day).beginning_of_day
+        end
+        # find next working range after time
+        time_in_day = time.seconds_since_midnight
+        time = time.beginning_of_day
```

Comment:
Doesn't the test at line 177 cover this scenario? Or did I miss something? (I think I just repeated the same reply!). Can you tell me how it fails and I'll re-submit?